README.md

```diff
@@ -228,236 +250,52 @@ messages about the conversion.
-the specified length. If a string longer than the specified length is stored,
-and the extra characters are all spaces, then it will be silently
-truncated. Moreover, trailing spaces are ignored when comparing two values. In
-constrast, Spanner does not give special treatment to spaces, and the specified
-length simply represents the maximum length that can be stored. This is close to
-the semantics of PostgreSQL's `VARCHAR(n)`. However there are some minor
-differences. For example, even `VARCHAR(n)` has some special treatment of
-spaces: strings longer than the specified length are silently truncated if the
-extra characters are all spaces.
-
-### Storage Use
-
-The tool maps several PostgreSQL types to Spanner types that use more storage.
-For example, `SMALLINT` is a two-byte integer, but it maps to Spanner's `INT64`,
-an eight-byte integer. This additional storage could be significant for large
-arrays.
-
-### Arrays
-
-Spanner does not support multi-dimensional arrays. So while `TEXT[4]` maps to
-`ARRAY<STRING(MAX)>` and `REAL ARRAY` maps to `ARRAY<FLOAT64>`, `TEXT[][]` maps
-to `STRING(MAX)`.
-
-Also note that PosgreSQL supports array limits, but the PostgreSQL
-implementation ignores them. Spanner does not support array size limits, but
-since they have no effect anyway, the tool just drops them.
-
-### Primary Keys
-
-Spanner requires primary keys for all tables. PostgreSQL recommends the use of
-primary keys for all tables, but does not enforce this. When converting a table
-without a primary key, HarbourBridge will create a new primary key of type
-INT64. By default, the name of the new column is `synth_id`. If there is already
-a column with that name, then a variation is used to avoid collisions.
-
-### NOT NULL Constraints
-
-The tool preserves `NOT NULL` constraints. Note that Spanner does not require
-primary key columns to be `NOT NULL`. However, in PostgreSQL, a primary key is a
-combination of `NOT NULL` and `UNIQUE`, and so primary key columns from
-PostgreSQL will be mapped to Spanner columns that are both primary keys and `NOT
-NULL`.
-
-### Foreign Keys and Default Values
-
-Spanner does not currently support foreign keys or default values. We drop these
-PostgreSQL features during conversion.
-
-### Other PostgreSQL features
-
-PostgreSQL has many other features we haven't discussed, including functions,
-sequences, procecdures, triggers, (non-primary) indexes and views. The tool does
-not support these and the relevant statements are dropped during schema
-conversion.
-
-See
-[Migrating from PostgreSQL to Cloud Spanner](https://cloud.google.com/spanner/docs/migrating-postgres-spanner)
-for a general discussion of PostgreSQL to Spanner migration issues.
-HarbourBridge follows most of the recommendations in that guide. The main
-difference is that we map a few more types to `STRING(MAX)`.
+Please refer below mentioned links for details on schema conversion of PostgreSQL
```

Comment:
Suggest replacing with:

Details on HarbourBridge schema conversion for PostgreSQL and MySQL can be found in the [PostgreSQL schema conversion](postgres/README.md#schema-conversion) and [MySQL schema conversion](mysql/README.md#schema-conversion) sections respectively.